c7n/resources/awslambda.py

```diff
@@ -255,6 +255,68 @@ class KmsFilter(KmsRelatedFilter):
     RelatedIdsExpression = 'KMSKeyArn'
 
 
+@AWSLambda.filter_registry.register('check-xray-tracing-enabled')
```

Comment:
we don't add dedicated filters for simple attribute checks, the value filter provides the ability to do this generically across any set of attributes.

```
 type: value
 key: TracingConfig.Mode
 value: Active
 op: not-equal
```

Reply:
Agreed. Removed filter code.